games-fps/shockolate/shockolate-20180924.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="System Shock, but cross platform!"
+DESCRIPTION="Based on the source code for PowerPC released by Night Dive Studios, Incorporated.\
+You need contents of data folder from the original game copied into res/data folder."
+HOMEPAGE="https://github.com/Interrupt/systemshock"
+COPYRIGHT="2018 Interrupt"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/Interrupt/systemshock/archive/1b1c664188ce736a343913de32afe627e48e7bbb.zip"
+CHECKSUM_SHA256="29685f42d4dc0e1b6a608ac61ce8f09fdce379784af73400fb9cd64aa9b6d51d"
+SOURCE_DIR="systemshock-1b1c664188ce736a343913de32afe627e48e7bbb"
+PATCHES="shockolate_x86-20180924.patchset"
```

Comment:
```diff
-PATCHES="shockolate_x86-20180924.patchset"
+PATCHES="shockolate-$portVersion.patchset"
```

Patches should be named with the name of the port, in this case is `shockolate`. It's also recommended to use `$portVersion` where possible.

Please rename the patch as well.